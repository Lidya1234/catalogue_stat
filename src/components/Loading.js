const Loading = () => (
  <div>
    <h1 className="load">Loading ...</h1>
  </div>
);
export default Loading;
