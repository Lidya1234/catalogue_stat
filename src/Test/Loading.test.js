import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Loading from '../components/Loading';
import '@testing-library/jest-dom/extend-expect';

describe('The Loading component should contain', () => {
  it('Loading text', () => {
    const { getByText } = render(
      <MemoryRouter>
        <Loading />
      </MemoryRouter>,
    );
    const text = getByText('Loading ...');
    expect(text).toBeInTheDocument();
  });

  const component = renderer.create(
    <MemoryRouter>
      <Loading />
    </MemoryRouter>,
  ).toJSON();

  test('Matchs snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
