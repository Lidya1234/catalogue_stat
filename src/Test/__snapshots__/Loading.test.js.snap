// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Loading component should contain Matchs snapshot 1`] = `
<div>
  <h1
    className="load"
  >
    Loading ...
  </h1>
</div>
`;

exports[`The navigation bar should contain Matchs snapshot 1`] = `
<div>
  <h1
    className="load"
  >
    Loading ...
  </h1>
</div>
`;
